src/pybind/mgr/dashboard/frontend/src/styles/ceph-custom/_icons.scss

```
@@ -1,10 +1,10 @@
 /* Icons */
 .ceph-icon {
-  background: url('../../assets/Ceph_Logo_Stacked_RGB_White_120411_fa_256x256.png');
+  background: url('assets/Ceph_Logo_Stacked_RGB_White_120411_fa_256x256.png');
```

We need to use relative paths.
Since we change all `@imports` and `@use` to use relative paths, I don't see a reason to try to not use it.
```suggestion
  background: url('../../assets/Ceph_Logo_Stacked_RGB_White_120411_fa_256x256.png');
```

---

Don't know what went wrong yesterday locally but now your suggested change works also locally.